model/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'java-library'
 apply plugin: 'com.google.protobuf'
+apply plugin: 'jacoco'
```

Comment:
we don't have any tests in the model module so we can remove jacoco from `model/build.gradle`.

Reply:
Yes taught i remove this , done